src/main/java/net/pms/dlna/RootFolder.java

```diff
@@ -97,11 +97,15 @@ private void addVirtualMyMusicFolder() {
 				LOGGER.debug("adding My Music folder to root");
 			}
 		} else {
-			if (PMS.get().getLibrary().getAudioFolder() != null && PMS.get().getLibrary().getAudioFolder().getChildren() != null &&
-				!PMS.get().getLibrary().getAudioFolder().getChildren().contains(myMusicFolder)) {
-				myMusicFolder.setFakeParentId(PMS.get().getLibrary().getAudioFolder().getId());
-				PMS.get().getLibrary().getAudioFolder().addChild(myMusicFolder, true, false);
-				LOGGER.debug("adding My Music folder to 'Audio' folder");
+			if (PMS.get().getLibrary() != null ) {
```

Comment:
Please delete the space before the close paren on this line to get pass checkstyle.